smartseq2_single_sample/pipelines/hisat2_QC_pipeline.wdl

```diff
@@ -17,8 +17,8 @@ workflow RunHisat2Pipeline {
   String sample_name
   ## variables to estimate disk size
   Float hisat2_ref_size = size(hisat2_ref,"GB")
-  Float fastq_size = size(fastq_read1,"GB") +size(fastq_read2,"GB")
-  Float reference_bundle_size = size(ref_fasta,"GB") + size(ref_flat,"GB")+size(rrna_interval,"GB")+size(gtf,"GB")
+  Float fastq_size = size(fastq_read1,"GB") + size(fastq_read2,"GB")
+  Float reference_bundle_size = size(ref_fasta,"GB") + size(ref_flat,"GB") + size(rrna_interval,"GB") + size(gtf,"GB")
   Float md_disk_multiplier = 10.0
   Int? increase_disk_size
```

Comment:
This `increase_disk_size` still confuses me. Could you explain what you're trying to accomplish to me? Maybe I just don't get it.